vyper/ir/README.md

```diff
@@ -1,10 +1,10 @@
-# Introduction to LLL
+# Introduction to IR
 
-LLL is Vyper's current intermediate representation. It is used as "a high level assembly". While LLL could theoretically compile to any backend which shares enough intrinsics with the EVM, this document assumes the reader has a good understanding of the EVM's execution model.
+IR is Vyper's current intermediate representation. It is used as "a high level assembly". While IR could theoretically compile to any backend which shares enough intrinsics with the EVM, this document assumes the reader has a good understanding of the EVM's execution model.
```

Comment:
"IR is Vyper's current intermediate representation"

This sentence is weird to read